### ISSUE TITLE
Loading a graph with saved layers enabled will cause an exception

### ISSUE THREAD

**Issue by aldebaran30701** [constellation-app/constellation#792]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Running the latest version of Constellation
	* Attached the ***Support Package*** via `Help` > `Support Package`
    * Checked the FAQs: https://github.com/constellation-app/constellation/wiki/FAQ
    * Checked that your issue isn't already filed: https://github.com/constellation-app/constellation/issues
    * Checked that there is not already a module that provides the described functionality: https://github.com/constellation-app/constellation/wiki/Catalogue-of-Repositories

### Description

Loading a graph which was saved when a layer was enabled will error. 

### Steps to Reproduce

New graph -> add elements -> open layers view - > add 1 element to layer 2 -> select layer 2 -> save graph
Close graph
Load graph
Exception occurs

**Expected behaviour:** loads graph with selection enabled

**Actual behaviour:** errors
```
java.lang.ArrayIndexOutOfBoundsException: Index -1107 out of bounds for length 16
	at au.gov.asd.tac.constellation.graph.StoreGraph.setObjectValue(StoreGraph.java:2549)
	at au.gov.asd.tac.constellation.graph.StoreGraph.recalculateLayerVisibilities(StoreGraph.java:1974)
	at au.gov.asd.tac.constellation.graph.StoreGraph.updateAllLayerMasks(StoreGraph.java:2223)
	at au.gov.asd.tac.constellation.graph.StoreGraph.setIntValue(StoreGraph.java:2346)
	at au.gov.asd.tac.constellation.graph.undo.UndoGraphEditOperation$17.execute(UndoGraphEditOperation.java:603)
	at au.gov.asd.tac.constellation.graph.undo.UndoGraphEditState.execute(UndoGraphEditState.java:435)
	at au.gov.asd.tac.constellation.graph.undo.UndoGraphEdit.execute(UndoGraphEdit.java:58)
	at au.gov.asd.tac.constellation.graph.locking.LockingManager$LockingEdit.execute(LockingManager.java:204)
	at au.gov.asd.tac.constellation.graph.locking.LockingManager$LockingEdit.commit(LockingManager.java:391)
	at au.gov.asd.tac.constellation.graph.locking.LockingManager.commit(LockingManager.java:153)
	at au.gov.asd.tac.constellation.graph.locking.LockingStoreGraph.commit(LockingStoreGraph.java:78)
	at au.gov.asd.tac.constellation.graph.locking.LockingStoreGraph.commit(LockingStoreGraph.java:73)
	at au.gov.asd.tac.constellation.graph.locking.LockingStoreGraph.commit(LockingStoreGraph.java:68)
	at au.gov.asd.tac.constellation.graph.interaction.plugins.clipboard.CopyToNewGraphPlugin.makeGraph(CopyToNewGraphPlugin.java:230)
	at au.gov.asd.tac.constellation.graph.interaction.plugins.clipboard.CopyToNewGraphPlugin.read(CopyToNewGraphPlugin.java:79)
[catch] at au.gov.asd.tac.constellation.plugins.templates.SimpleReadPlugin.run(SimpleReadPlugin.java:81)
	at au.gov.asd.tac.constellation.graph.node.plugins.DefaultPluginEnvironment.lambda$executePluginLater$0(DefaultPluginEnvironment.java:129)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

```

**Reproduces how often:** 100%

### Additional Information

This does not occur if you do not have the layer enabled when you save.

**Comment by aldebaran30701** [comment]
@arcturus2  @antares1470 
Just going through the board, I did some testing and this issue appears to be fixed from #849 
I'd say close it or possibly also take 2 minutes to test this to confirm it is also fixed.

**Comment by antares1470** [comment]
> @arcturus2 @antares1470
> Just going through the board, I did some testing and this issue appears to be fixed from #849
> I'd say close it or possibly also take 2 minutes to test this to confirm it is also fixed.

@aldebaran30701 I've just given this a test and it would appear that this is fixed so I'll close this.
